app/models/changeset/pull_request.rb

```diff
@@ -3,8 +3,9 @@ class Changeset::PullRequest
   CODE_ONLY = "[A-Z][A-Z\\d]+-\\d+"  # e.g., S4MS0N-123, SAM-456
   PUNCT = "\\s|\\p{Punct}|~|="
 
-  # Matches a section heading named "Risks".
+  # Matches a markdown section heading named "Risks".
   RISKS_SECTION = /#+\s+Risks.*\n/i
+  RISKS_SECTION_ALT = /^\s*Risks.*\n=+\s*$/mi
```

Comment:
so this ... `Risks` or `Risk` with `#` in front of it and optionally followed by heading `=`(h1) or `-`(h2)
`/^\s*#*Risks?\s*\n(?:^\s*:[=-]+\s*$)?/mi`